2020-12-01 09:07:59 ERROR {FuzzyMatch} [main] File or Directory not found
Traceback (most recent call last):
  File "C:\Users\Javier Landin\source\repos\PYTHON\FuzzyMatch\FuzzyMatch.py", line 25, in main
    processFiles(Path(chItem + "sdaasd"))
  File "C:\Users\Javier Landin\source\repos\PYTHON\FuzzyMatch\FuzzyMatchFuncs.py", line 25, in processFiles
    for itemFile in os.listdir(directory):
FileNotFoundError: [WinError 3] El sistema no puede encontrar la ruta especificada: '\\\\SGC\\ML_Project\\MLM_1\\MLM1953\\Transformsdaasd'
